tests/features/consultation.feature

```diff
@@ -10,6 +10,11 @@ Feature: Consultation content creation
     And the following document:
       | name          | file       |
       | My Document 1 | sample.pdf |
+    And the following document:
+      | name          | file         |
```

Comment:
This can be removed and the document would be created by the previous step.

Reply:
I need an another document because `sample.pdf` is printed also on the page. We check:
```
And I should see "sample.pdf"
And I should see "document.pdf"
```